task/net/src/bsp/sidecar_1.rs

```diff
@@ -0,0 +1,244 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::GPIO;
+use drv_spi_api::Spi;
+use drv_stm32h7_eth as eth;
+use drv_stm32h7_gpio_api as gpio_api;
+use ksz8463::{Ksz8463, Register as KszRegister};
+use ringbuf::*;
+use userlib::{hl::sleep_for, task_slot};
+use vsc7448_pac::types::PhyRegisterAddress;
+use vsc85xx::{Phy, PhyRw, PhyVsc85xx, VscError};
+
+task_slot!(SPI, spi_driver);
+const KSZ8463_SPI_DEVICE: u8 = 0; // Based on app.toml ordering
```

Comment:
Yeah, sorry about this, I need to finish the code to generate a named enumeration. :-\